crates/sui-bridge-indexer/src/postgres_manager.rs

```diff
@@ -42,3 +46,29 @@ pub fn write(pool: &PgPool, token_txns: Vec<TokenTransfer>) -> Result<(), anyhow
     })?;
     Ok(())
 }
+
+pub fn get_newest_finalized_token_transfer(
+    pool: &PgPool,
+) -> Result<Option<DBTokenTransfer>, Error> {
+    use crate::schema::token_transfer::dsl::*;
+
+    let connection = &mut pool.get().unwrap();
+    token_transfer
+        .filter(status.ne("DepositedUnfinalized"))
```

Comment:
typo - this is why we need unit tests :)

Reply:
wym here?

Reply:
will do